stripper/ze_legoland_crackheads_p2.cfg

```diff
@@ -0,0 +1,27 @@
+;Fix boss from dieing on different endings which can cause a nuke bug
+modify:
+{
+	match:
+	{
+		"classname" "math_counter"
+		"targetname" "ColorCounter"
+	}
+	replace:
+	{
+		"startvalue" "2250"
```

Comment:
Raising the initial boss HP is a stupid way to fix this. You could instead do something like remove the output on ColorBoss, and only add it back when it's time for the boss fight.